README.md

```diff
@@ -29,15 +29,66 @@ $ npm install
 
 ## Running the app
 
-```bash
-# development
-$ npm run start
+- Using Docker
+  - `docker-compose up` (first time run will perform build)
+    - you can force a fresh build with `--build`
+    - you can background (run without holding up your CLI) by using `--detach`
+- Without Docker
+
+  ```bash
+    # development
+    $ npm run start
+
+    # watch mode
+    $ npm run start:dev
+
+    # production mode
+    $ npm run start:prod
+  ```
 
-# watch mode
-$ npm run start:dev
+## Publishing Messages
+
+To publish events to the org-creator service to process run this snippet as a ts or js file.
```

Comment:
Would be good to add what the expected output of this script is

Reply:
I am not sure about this. Since this is just publishing events for testing purpose and the events are processed asynchronously and no immediate acknowledgment , it doesn't return an output. I can't also think of a better way to test it without the FE part of the work being completed. what do you suggest.

Reply:
Even if async, would we still expect logs from the server indicating that message is received within a few seconds? I think example expected output logs from the server would be helpful here. If no server doesn't write logs, even adding a note that this server will handle this request silently would be valuable information someone running this code (though not that helpful a test in this case, IMO).

If the server doesn't write any logs upon receiving this request, maybe we should make a simple `health` endpoint that can be called or a simple "health" message that gets published to nats to test the nats connection? I think that is not as good as testing the actual server functionality as you have done but maybe it is an option.